next.config.js

```diff
@@ -30,6 +30,10 @@ const moduleExports = {
   pageExtensions: ['tsx', 'js', 'jsx', 'mdx', 'ts'],
   images: {
     domains: ['cdn.discordapp.com']
+  },
```

Comment:
what does this do?

Reply:
It's used to config the languages the app will support and set a default one that will be used for all the routing.